test/e2e/app-dir/parallel-routes-layouts/parallel-routes-layouts.test.ts

```diff
@@ -0,0 +1,84 @@
+import { createNextDescribe } from 'e2e-utils'
+import { retry } from 'next-test-utils'
+
+createNextDescribe(
+  'parallel-routes-layouts',
+  {
+    files: __dirname,
+  },
+  ({ next }) => {
```

Comment:
```suggestion
describe('parallel-routes-layouts', () => {
  const { next } = nextTestSetup({
    files: __dirname,
  })
```

Reply:
sgtm, but we should probably discuss with the rest of the team as well since I think most tests are written this way currently. It seems marginally useful for things like `skipStart` and `skipDeployment` but honestly doesn't save that much time so 🤷

Reply:
those are also included in `nextTestSetup` if you look at the implementation `createNextDescribe` is just a wrapper (I changed that a while back after talking to the team)

Reply:
Oh nice! Ok I can update this.